js/views/onboardingModal.js

```diff
@@ -0,0 +1,469 @@
+'use strict';
+
+var Backbone = require('backbone'),
+    loadTemplate = require('../utils/loadTemplate'),
+    baseModal = require('./baseModal'),
+    messageModal = require('../utils/messageModal.js'),
+    countryListView = require('../views/countryListVw'),
+    currencyListView = require('../views/currencyListVw'),
+    languageListView = require('../views/languageListVw'),
+    guidStillCreatingModal = require('../views/guidStillCreatingModal');
```

Comment:
You will need to add cropit too
cropit = require('../utils/jquery.cropit'),

It's used by the avatar uploader.